sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -418,3 +425,166 @@ def analyze_sentiment(  # type: ignore
             )
         except HttpResponseError as error:
             process_batch_error(error)
+
+    @distributed_trace
+    def get_healthcare_results(  # type: ignore
```

Comment:
I added the operation IDs for all of the endpoints except `health`. Once the service adds the operation IDs to the endpoint, the names will probably be different